iothub/service/src/DigitalTwin/Models/DigitalTwinUpdateResponse.cs

```diff
@@ -34,6 +34,6 @@ protected internal DigitalTwinUpdateResponse(ETag eTag = default, string locatio
         /// <summary>
         /// Gets the URI of the digital twin.
         /// </summary>
-        public string Location { get; }
+        internal string Location { get; }
```

Comment:
Please add a comment as to why we're hiding this.

1. No known use case for users.
2. Only added to the service to be complete with guidance.